packages/protocol/README.md

```diff
@@ -149,10 +146,8 @@ Even though this decoding operation overhead will recur a lot, it is still negli
 The address of the default BIP 44 wallet (`m/44'/60'/0'/0/0`) derived from this seed is used to identify the Airnode.
 The Airnode (referring to the node application) polls `AirnodeRrp` for `MadeTemplateRequest` and `MadeFullRequest` events indexed by its own identifying address (and drops the ones that have matching `FulfilledRequest` and `FailedRequest` events).
 
-Note that the Airnode operator may not announce its extended public key (that belongs to the HDNode with the path `m/44'/60'/0'`) on-chain, in which case it would have to do it through off-chain channels.
-
 2. A developer decides to build a contract that makes requests to a specific Airnode (we will call this contract *requester*).
-Using the `xpub` of the Airnode and the address of an Ethereum account they control, the developer derives the address of their sponsor wallet (see below for how this is done).
+Using the `xpub` of the Airnode (which is announced off-chain) and the address of an Ethereum account they control, the developer derives the address of their sponsor wallet (see below for how this is done).
```

Comment:
maybe specify xpub like 
> Using the `xpub` (extended public key) of the Airnode            


because I dont see it being mentioned anywhere what xpub is